src/components/TokenWarningModal/index.tsx

```diff
@@ -55,10 +56,13 @@ interface TokenWarningCardProps {
 
 export function TokenWarningCard({ token, list }: TokenWarningCardProps) {
   const theme = useTheme()
+  const { t } = useTranslation()
   const { chainId } = useActiveWeb3React()
 
   if (!token) return null
 
+  const externalLinkText = t('viewOnBlockExplorer')
```

Comment:
same, I don't think we need a variable for the translation

Reply:
done